build.ps1

```diff
@@ -898,6 +932,41 @@ function Build-SourceKitLSP($Arch)
     }
 }
 
+function Build-Installer()
+{
+  Build-WiXProject toolchain.wixproj -Properties @{
+    DEVTOOLS_ROOT = "$ToolchainInstallRoot\";
+    TOOLCHAIN_ROOT = "$ToolchainInstallRoot\";
+  }
+
+  msbuild $SourceCache\swift-installer-scripts\platforms\Windows\CustomActions\SwiftInstaller\SwiftInstaller.vcxproj -t:restore
+  Build-WiXProject sdk.wixproj -Properties @{
+    PlatformToolset = "v143";
+    PLATFORM_ROOT = "$PlatformInstallRoot\";
+    SDK_ROOT = "$SDKInstallRoot\";
+    SWIFT_SOURCE_DIR = "$SourceCache\swift\";
+  }
+
+  Build-WiXProject runtime.wixproj -Properties @{
+    SDK_ROOT = "$SDKInstallRoot\";
+  }
+
+  Build-WiXProject icu.wixproj -Properties @{
+    ProductVersion = "69.1";
+    ProductVersionMajor = "69";
+    ICU_ROOT = "S:\";
+  }
```

Comment:
This should no longer be needed on 5.8+ as ICU is statically linked into Foundation.